Library/Homebrew/cmd/audit.rb

```diff
@@ -512,9 +512,9 @@ def audit_github_repository
     end
 
     problem "GitHub fork (not canonical repository)" if metadata["fork"]
-    if (metadata["forks_count"] < 10) && (metadata["subscribers_count"] < 10) &&
-       (metadata["stargazers_count"] < 20)
-      problem "GitHub repository not notable enough (<10 forks, <10 watchers and <20 stars)"
+    if (metadata["forks_count"] < 50) && (metadata["subscribers_count"] < 50) &&
+       (metadata["stargazers_count"] < 100)
```

Comment:
> I don't want Homebrew to be the source for a bunch of weekend projects.

I agree. It's very easy to create a tap and the installation is still seamless, given that Homebrew automatically taps when a fully qualified formula name is provided. The main thing inclusion in core gives them is visibility. This can wait until a project has matured and has a substantial user base.

FWIW, I'm neutral towards this change. Just wanted to share my thoughts.